packages/ember-data/lib/system/model/states.js

```diff
@@ -399,6 +399,9 @@ var createdState = dirtyState({
   isNew: true
 });
 
+createdState.invalid.rolledBack = function(record) {
```

Comment:
This is much better solution then my last commit. I was trying to handle it same way but within **createdState.invalid.rolleback** which is of course wrong. +1